app/tut/ga/usaideach.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Roinnt Gníomhartha Úsáideacha
+next: cluiche
+next-text: Cluiche
+prev: gniomh
+prev-text: Ní Briathra a Dhearbhaíonn Ach Gníomh
+tsn-fname: useful
+---
+
+# Mata
+
+Go dtí seo chonaiceamar go leor gníomhartha difriúla (`codladh`{.setanta}, `scríobh`{.setanta}
+srl.). Sula leanaimid ar aghaidh le níos mó [[gnéithe|features]] *Setanta*, ba chóir dúinn fanacht
+ar feadh nóiméad agus féachaint ar cúpla gníomhartha agus oibritheoirí úsáideacha eile.
+
+## Níos mó oibritheoirí
+
+Chonaiceamar níos luaithe sa teagasc seo na hoibritheoirí `+`, `-`, `*` agus `/`. Tá dhá
+oibritheoirí matamaitice eile ag *Setanta*, an oibritheoir "modulo" (`%`) agus an oibritheoir
+[[roinnt-slánuimhreacha|integer division]] (`//`).
+
+### Modulo
+
+Is oibritheoir [[neamhghnách|unusual]] é an oibritheoir modulo (`%`), ach tá sé an-úsáideach.
+
+Glacann an oibritheoir sin le dhá uimhir, [[roinneann sé|it divides]] an chéad ceann leis an dara
+agus tugann sé ar ais an [[fuílleach|remainder]].
+
+Mar shampla: tá `7 % 2`{.setanta} cothrom le `1`{.setanta} mar nuair a roinneann tú `7`{.setanta} le
+`2`{.setanta}, tá an fuílleach cothrom le `1`{.setanta} (mar tá `7 == 3*2 + 1`{.setanta}). Mar an
+gcéanna, tá `11 % 4`{.setanta} cothrom le `3`{.setanta} mar tá `11 == 2*4 + 3`{.setanta}.
+
+#### Corr nó Réidh?
+
+Tá an oibritheoir seo úsáideach mar is féidir linn é a úsáid le haghaidh a lán rudaí a dhéanamh. Mar
+shampla: is féidir linn é a úsáid chun seiceáil an bhfuil uimhir éigin [[corr|odd]] nó
+[[réidh|even]]. Chun é sin a dhéanamh níl ach le déanamh againn ach seiceáil cad é an fuílleach
+nuair a roinneann tú an uimhir faoi 2. Má tá an fuílleach cothrom le 0, is uimhir réidh é mar is
+féidir linn an uimhir a roinn faoi 2 gan fuílleach. Má tá sé cothrom le 1 is uimhir corr é. Bain
+triail as an ríomhchlár seo:
+
+{{{
+le i idir (0, 10) {
+    má i % 2 == 0 {
+        scríobh("Is uimhir réidh é", i)
+    } nó {
+        scríobh("Is uimhir corr é", i)
+    }
+}
+}}}
+
+#### An bhfuil sé [[príomha|prime]]?
+
+Is féidir linn an oibritheoir modulo `%` a úsáid chun seic a dhéanamh an féidir leat uimhir a roinnt
+faoi uimhir eile (má tá `x % y == 0`{.setanta}, is féidir leat `x` a roinn faoi `y`). Is féidir linn
```

Comment:
a roinnt ar (tá mé nach mór cinnte gur roinntear x ar y seachas faoi y nuair is x/y atá i gceist)

Reply:
Athraithe